Dockerfile

```diff
@@ -11,14 +11,13 @@ COPY package.json yarn.lock ./
 
 # install dependencies
 RUN yarn install --pure-lockfile
-RUN yarn global add gulp-cli@2.2.1
 
 # copy supporting files and media
-COPY .eslintrc.js .eslintignore .stylelintrc .stylelintignore gulpfile.js ./
+COPY .eslintrc.js .eslintignore .stylelintrc webpack.config.js webpack.static.config.js ./
```

Comment:
do we no longer need a `.stylelintignore` ?

Reply:
oh, yes we do - good catch!